README.md

```diff
@@ -61,6 +61,34 @@ class TestCustomerModel(TestCase):
 
 Check out [documentation](<http://model-bakery.readthedocs.org/>) for more complete examples.
 
+## Contributing
+
+1. Fork this repo and clone it to your computer:
+
+```
+$ git clone git@github.com:YOUR_USER/model_bakery.git
+```
+
+2. Install the dev dependencies:
+
+```
+$ pip install -r dev_requirements.txt
```

Comment:
I think that would be nice to remove `$` from these lines do make easier for people to copy/paste.

Reply:
Fixed by f78f9cc77e4c406682169dfd64033dde0b6da707